apps/math_variable_box_empty_controller.cpp

```diff
@@ -5,7 +5,7 @@
 
 MathVariableBoxEmptyController::MathVariableBoxEmptyView::MathVariableBoxEmptyView() :
   ModalViewEmptyView(),
-  m_layoutExample(0.5f, 0.5f, KDColorBlack, Palette::WallScreen)
+  m_layoutExample(0.5f, 0.5f)
```

Comment:
I know this work (thanks to the default values of the parent), but I would leave the parameters so as not to change too much what Numworks did and to facilitate merges with upstream. So, do something like this :

```suggestion
  m_layoutExample(0.5f, 0.5f)
  m_layoutExample(0.5f, 0.5f, Palette::..., Palette::...)
```

Furthermore, if you change this here, you have to change this everywhere in Epsilon (aka. all over)...

Note: this review can be applied to several lines of your PR, that I have not listed here